dm/pkg/ha/bound.go

```diff
@@ -202,46 +202,86 @@ func GetLastSourceBounds(cli *clientv3.Client) (map[string]SourceBound, int64, e
 // if source bound is empty, will return an empty sourceBound and an empty source config
```

Comment:
in the comment "The index worker **must not be empty**:". What's the meaning of index worker?

Reply:
i think it‘s the parameter `worker` the same as it in master branch.